third_party/payjs/third_party/web_activities/README.md

```diff
@@ -0,0 +1,11 @@
+# Web Activities JavaScript library
+
+This is an import of the OSS [Web Activities](https://github.com/google/web-activities) library.
```

Comment:
I think we used to have a third_party fork of web-activities but now we import it.

Reply:
third party changes cl https://github.com/ampproject/amphtml/pull/24871